SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -30,12 +32,11 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 public:
 	UAbstractLBStrategy();
 
-	virtual void Init(const USpatialNetDriver* InNetDriver);
+	virtual void Init(const USpatialNetDriver* InNetDriver) {}
```

Comment:
What's the justification for passing in the NetDriver here? Overusing the NetDriver for things seems to be a reason we've got situations where we're avoiding things like `check(NetDriver)` because we can't mock it in tests, so usage should aim be as sparing as possible?

Reply:
In the GridBased one, currently we just call `super::init(inNetDriver)` then nothing else

Reply:
It might be something we can remove at this point, at least for the single use case of the GridBased, but originally we were thinking it would be the way the Strategy would get access to the Translator. We may want to rethink that, but I'd rather wait for the code to be a bit more gelled, since we seem to be swapping back and forth with every PR.